doc-content/jbpm-docs/src/main/asciidoc/BPMN2/copy-elements-proc.adoc

```diff
@@ -0,0 +1,19 @@
+[id='copy-elements']
+
+= Copying elements
+You can copy individual elements and finished business processes.
+
+.Procedure
+
+. On the canvas, click and drag the cursor to select the elements you want to copy.
+. Click image:processes/3417.png[] to copy your selection.
+. Switch into the second business process where you want to your add the copied elements.
```

Comment:
well given this text I understand the goal is to copy some elements from a business process A, and then copy them into another porcess B, but in fact this feature will be not ready until 7.5.GA. Actually only copy/pasting elements in the same process is allowed, not between processes